.travis.yml

```diff
@@ -4,12 +4,12 @@ env:
     - JRUBY_OPTS="$JRUBY_OPTS --debug"
 language: ruby
 rvm:
-  - 1.9.3
   - 2.0.0
   - 2.1
   - 2.2
   - 2.3.0
-  - jruby
+  - jruby-9.0.0.0
+  - jruby-9.0.1.0
```

Comment:
Added 9.0.{4,5}.0 as well... But there are couple of specs failing on 9000+ in general o_O